scripts/validate-urls.py

```diff
@@ -1,4 +1,22 @@
-# This script extract http/https URLs from .md files and validate them.
+# This script finds .md files under a directory and its subdirectories, extracts
+# http/https URLs from .md files and validates them.
+#
+# This script can be run peoriodically on kubeflow/website source repository
+# to find outdated URLs, which indicate possible outdated document sections.
+#
+# To run this script, type
+#   $ python3.8 validate-urls.py -d /path/to/kubeflow/website/content/docs
```

Comment:
I think it'd be better to remove the $ sign. People may think they need to include it. Maybe say something like this:

```
# To run this script, type the following on the command line:
# python3.8 validate-urls.py -d /path/to/kubeflow/website/content/docs
```